File: CivitasJ/src/civitas/OperacionInmobiliaria.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package civitas;

/**
 *
 * @author ELVIRA
 */
public class OperacionInmobiliaria {
    private int numPropiedad;
    private GestionesInnmobiliarias gestion;

    public OperacionInmobiliaria(GestionesInnmobiliarias gestion, int numPropiedad) {
        this.numPropiedad = numPropiedad;
        this.gestion = gestion;
    }

    public int getNumPropiedad() {
        return numPropiedad;
    }

    public GestionesInnmobiliarias getGestion() {
        return gestion;
    }
    
    
}

File: CivitasR/lib/civitas/operacion_inmobiliaria.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

class OperacionInmobiliaria
  
  attr_reader :numPropiedad, :gestion
  
  
  def initialize (gestion, numPropiedad)
    @numPropiedad = numPropiedad
    @gestion = gestion
  end
  
  
end
